### DISCUSSION TITLE
Add disjunctiveNumericRefinements

### DISCUSSION THREAD

**Discussion by JanPetr** [algolia/instantsearch#5740] [Ideas]
Currently there is no way how to add disjunctive numeric refinements so it need to be bypassed by something like:

```
helper.setQueryParameter('filters', 'price = -1 OR expires >= 50');
```

And in the time when helper starts to use `filters` query parameter the code might break.

Customer's issue: http://stackoverflow.com/questions/40977909/algolia-how-do-i-add-is-it-possible-to-add-a-numeric-disjunctive-refinement/40979372

**Comment by seafoox** [comment]
Definitely something we need 👍

**Comment by Jerska** [comment]
Had the case just today 👍 Or at least to plan migrating to `filters` as a breaking change of the helper.

**Comment by Haroenv** [comment]
Something we needed yesterday too for date ranges (in the end it worked with AND as well, but would still be nice), cc @samouss

**Comment by Haroenv** [comment]
Just had another use case for this
